src/solidity/astHelper.js

```diff
@@ -59,14 +59,31 @@ function extractState (contractName, sourcesList) {
       }
     }
     return {
-      stateItems: stateItems,
+      stateDefinitions: stateItems,
       stateVariables: stateVar
     }
   }
   return null
 }
 
+/**
+  * return state var and type definition of all the contracts from the given @args sourcesList
+  *
+  * @param {Object} sourcesList - sources list (containing root AST node)
+  * @return {Object} - returns a mapping between contract name and contract state
+  */
+function extractStates (sourcesList) {
+  var contracts = extractContractDefinitions(sourcesList)
```

Comment:
This is called again inside `extractState` - can you make it an optional parameter for `extractState`?